MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -945,7 +945,12 @@ protected override string GetVideoArguments(StreamState state, EncodingOptions e
 
                 var hasGraphicalSubs = state.SubtitleStream != null && !state.SubtitleStream.IsTextSubtitleStream && state.SubtitleDeliveryMethod == SubtitleDeliveryMethod.Encode;
 
-                args += " " + EncodingHelper.GetVideoQualityParam(state, codec, encodingOptions, GetDefaultEncoderPreset()) + keyFrameArg;
+                args += " " + EncodingHelper.GetVideoQualityParam(state, codec, encodingOptions, GetDefaultEncoderPreset());
+
+                // Unable to force key frames to h264_qsv transcode
+                if (codec != "h264_qsv") {
+                    args += " " + keyFrameArg; 
+                }
```

Comment:
there should be a debug logging saying "cannot force keyframes" or something similar so we won't lose it

Reply:
How do you log in jellyfin?

Reply:
Worked it out for myself. Done.